articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -146,18 +150,17 @@ namespace TestConsole
             string serviceBusTopic = Environment.GetEnvironmentVariable(ServiceBusTopicEnvVarName);
             string serviceBusSubscription = Environment.GetEnvironmentVariable(ServiceBusSubscriptionEnvVarName);
             SubscriptionClient serviceBusClient = new SubscriptionClient(serviceBusConnectionString, serviceBusTopic, serviceBusSubscription);
-
             serviceBusClient.RegisterMessageHandler(
                 handler: (message, cancellationToken) =>
-               {
-                   string messageText = Encoding.UTF8.GetString(message.Body);
-                   JsonElement messageData = JsonDocument.Parse(messageText).RootElement.GetProperty("data");
-                   string key = messageData.GetProperty("key").GetString();
-                   Console.WriteLine($"Event received for Key = {key}");
-
-                   _refresher.SetDirty();
-                   return Task.CompletedTask;
-               },
+                {
+                    EventGridEvent eventGridEvent = EventGridEvent.Parse(BinaryData.FromBytes(message.Body));
+
+                    eventGridEvent.TryCreatePushNotification(out PushNotification pushNotification);
```

Comment:
It will be helpful to have some comments. It has two benefits
1. Help readers to understand the code
2. Attract readers' attention to what matters

Reply:
Good idea. I added comments